openwisp_radius/api/freeradius_views.py

```diff
@@ -445,6 +448,11 @@ def _data_to_acct_model(self, valid_data):
         valid_data['organization'] = acct_org
         return valid_data
 
+    def send_radius_accounting_signal(self, accounting_data):
+        radius_accounting_success.send(
+            sender=self.__class__, accounting_data=accounting_data
```

Comment:
The sender is the view, I noticed only now, it makes sense, but we haven't specified this in the docs, so users would have to dig in the code to find out, which is means the documentation is not good enough. Please fix this.

Reply:
I thought that it could be useful to pass the view instance (self) instead of the class, because that allows accessing a lot of information about the request and it can be useful to developers who may want to use this signal. Can you please update it here and in the docs?